src/main/resources/db/changelog/v.0.1/db.changelog-users.yaml

```diff
@@ -0,0 +1,58 @@
+databaseChangeLog:
+  - changeSet:
+      id: 10-add-user-role-enum
+      author: elizaveta_dubogryz
+      dbms: postgresql
+      sql: CREATE TYPE USER_ROLE as ENUM ('USER', 'ADMIN');
```

Comment:
I believe you have 3 roles, aren't you?

Reply:
We finally decided with frontend team that we'll have 2 roles

Reply:
Also this way we'll allow to separate the role filtering ("findPeopleByRole endpointor" find "findAllUsers") from "findAllPeople search".
More over it's a good style - to name diferrent entityes with different names...

Reply:
Not critical, but could be renamed. You should decide it inside your team

Reply:
About 2 roles lets discuss on the status call